client/blocks/video-editor/index.jsx

```diff
@@ -42,19 +41,9 @@ class VideoEditor extends Component {
 		pauseVideo: false,
 	};
 
-	// @TODO: Please update https://github.com/Automattic/wp-calypso/issues/58453 if you are refactoring away from UNSAFE_* lifecycle methods!
-	UNSAFE_componentWillReceiveProps( nextProps ) {
-		if ( nextProps.shouldShowError && ! this.props.shouldShowError ) {
-			this.setState( {
-				error: true,
-				pauseVideo: false,
```

Comment:
Can't see a good reason (or reproduce) why we might have wanted to set `pauseVideo` to `false` on error, but wanted to note that the change makes sense to me.